platform/accessibility/508-accessibility-best-practices.md

```diff
@@ -35,22 +42,26 @@
+- Going forward, we should also be testing pages with [axe-coconut](https://www.deque.com/blog/test-leading-edge-accessibility-axe-coconut-axe-core-3-0/), which identifies experimental WCAG2 warnings and best practices.
+- Run the [axe-end to end script with CSV and console output](https://github.com/adhocteam/a11y-snippets/tree/master/test-scripts/axe-e2e) for automated scans of high page count static sites. Errors can be outputted to CSV for easy spreadsheet import.
+- Developers should run the axe-core plugin while developing new components. Many automated errors can be avoided or fixed during the development cycle.
+- Validate rendered HTML to avoid automated a11y errors
+
+- **Automated build process** When you push your code to a feature branch or merge to master, the automated build process will test for accessibility/508 compliance on all rendered pages. The axe check scans for Section 508, WCAG2 A and WCAG2 AA [ruleset](https://dequeuniversity.com/rules/axe/) violations.
+    - **Each rendered page must pass an axe check.**
       * Static Markdown pages should be checked for violations using the axe plugin.
+      * Static content created by Markdown files or the Content Management System (CMS) will be tested as part of the build step.
       * Pages created with the content management system (CMS) should also be checked using the axe plugin.
-      * Client-side applications **must include an axe check** in their end-to-end tests. The engineering team has created a [Nightwatch helper function](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/testing/e2e/nightwatch-commands/axeCheck.js) for this purpose.
-
-2. When you push your code to a feature branch or merge to master, the automated build process will test for Accessibility/508 compliance.
-   1. Static content created by Markdown files or the Content Management System (CMS) will be tested as part of the build step.
-   2. Client-side applications will be tested for accessibility as part of their larger e2e test suite.
+      * Client-side applications will be tested for accessibility as part of their larger e2e test suite.
+      * Client-side applications **must include an axe check** in their end-to-end tests. The engineering team has created a [Nightwatch helper function](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/testing/e2e/nightwatch-commands/axeCheck.js) for this purpose
+      * ESLint exists as a Node module with a command-line interface. It runs a lint scan on all components as a pre-commit hook, and will break on errors.
+      * Generally speaking, ESLint and the needed dependencies should be installed when users *yarn install* from the *vets-website* root.
+      * If a build error occurs, fix the issue and submit your code again.
 
-3. If a build error occurs, fix the issue and submit your code again.
-
-## Manual Accessibility/508 Testing
+## Manual Accessibility Testing
```

Comment:
The table of contents link at the top of the page needs to be updated so this works properly. The text should match this heading, and the link link will look like this:

```html
[Manual Accessibility Testing](#manual-accessibility-testing)
```